### PR TITLE
DEV-3198-new-go-exercise-get-digit-len-fix

### PR DESCRIPTION
**Pull request by tiagocollot**
Added README.md for Go exam digitlen.

### CODE REVIEW

File: subjects/digitlen/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## digitlen
+
+### Instructions
+
+Write a function `DigitLen()` that takes two integers as arguments and returns the number of digits in the first `int` by the base of the second `int`.
```
Comment (jotapero) [inline subjects/digitlen/README.md:5]: Fix: (instructions are not that clear) "Write a function `DigitLen()` that takes two integers as arguments and returns the times the first `int` can be divided by the second until it reaches zero"

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/digitlen/README.md (hunk 1 above)
Comment (tiagocollot) [reply subjects/digitlen/README.md:5]: Thank you José. It is fixed.

**Review by jotapero — APPROVED** [review]
LGTM :alien:

File: subjects/digitlen/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+## digitlen
+
+### Instructions
+
+Write a function `DigitLen()` that takes two integers as arguments and returns the times the first `int` can be divided by the second until it reaches zero.
+
+- The second `int` must be between ***2*** and ***36***. If not, the function returns `-1`.
```
Comment (eslopfer) [inline subjects/digitlen/README.md:7]: When I ran prettier it complained here.

**Review by eslopfer — CHANGES_REQUESTED** [review]

**Pull request merged**